docker-compose.yml

```diff
@@ -9,3 +9,4 @@ services:
       - .env.secret
     extra_hosts:
       - "host.docker.internal:host-gateway"
+    privileged: true
```

Comment:
Is there a reason why the container needs/must be privileged?

Reply:
This relates to the mem_max and I'll try to address this and it together.

Reply:
I replaced this with 
```
    cap_add:
      - NET_ADMIN
```

that gives cloudflared the ability to increase its receive buffer. resolving the warning cloudflared was giving:

> 2025/06/28 18:52:26 failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 7168 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details.

The cap_add is much more appropriate than the blanket privileged.

Refs: 
* https://medium.com/mercedes-benz-techinnovation-blog/linux-capability-net-admin-in-a-hardened-kubernetes-world-216fc1a7be3
* https://docs.docker.com/reference/compose-file/services/#cap_add
* https://man7.org/linux/man-pages/man7/capabilities.7.html